test/jdk/java/awt/List/ListEnterExitTest.java

```diff
@@ -36,15 +36,22 @@
 import java.awt.Point;
 import java.awt.Robot;
 
-import java.awt.event.InputEvent;
 import java.awt.event.MouseAdapter;
 import java.awt.event.MouseEvent;
 
+import java.util.concurrent.CountDownLatch;
+import java.util.concurrent.TimeUnit;
+
 public class ListEnterExitTest {
     final List list = new List();
-    final MouseEnterExitListener mouseEnterExitListener = new MouseEnterExitListener();
     Frame frame;
     volatile Point p;
+    private static final int X_OFFSET = 30;
```

Comment:
```suggestion
    volatile Point p;

    private static final int X_OFFSET = 30;
```
I'd add a blank line between these two blocks.